README.md

```diff
@@ -41,17 +41,13 @@ First install the required dependencies through `bundle install`.
 
 The project is configured through environment variables.
 Copy the `.env.sample` configuration file to `.env`.
```

Comment:
Absolutely. As I mentioned in my previous comment, this is forked code that needs to be cleaned up eventually. For now, we just wanted to fork and see if this is something we could use for our dashboard. Then, I started noticing how much cleanup was necessary. I think eventually I might start over with a fresh repo since I've changed so much code.